src/modules/ie-objects/hooks/get-ie-objects-info.ts

```diff
@@ -16,11 +17,28 @@ export const useGetIeObjectInfo = (
 		enabled?: boolean;
 	} = {}
 ): UseQueryResult<IeObject | null> => {
-	return useQuery([QUERY_KEYS.getIeObjectsInfo, id], () => getIeObjectInfo(id), {
-		keepPreviousData: true,
-		enabled: true,
-		...options,
-	});
+	const router = useRouter();
+	return useQuery(
+		[QUERY_KEYS.getIeObjectsInfo, id],
+		async () => {
+			let newId;
+			if (id.length > 36) {
+				const v3Identifier = await IeObjectsService.schemaIdentifierLookup(id);
+				newId = v3Identifier.schemaIdentifierV3;
```

Comment:
When i read "v3identifier" i expect it to contain a string. This naming is better:
```suggestion
				const v3IdentifierResponse = await IeObjectsService.schemaIdentifierLookup(id);
				newId = v3IdentifierResponse.schemaIdentifierV3;
```